bindings/dotnet/build.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser
+from pathlib import Path
+import shutil
+import subprocess
+
+
+def classifier_to_target(classifier: str) -> str:
```

Comment:
TODO - properly distribute

Perhaps refer to librdkafka.redist -

- https://github.com/confluentinc/librdkafka/wiki/librdkafka.redist
- https://github.com/confluentinc/librdkafka/blob/master/packaging/nuget/templates/librdkafka.redist.props